src/fast_check/mod.rs

```diff
@@ -119,23 +147,25 @@ impl FastCheckDiagnostic {
       UnsupportedTsExportAssignment { range } => Some(range),
       UnsupportedTsNamespaceExport { range } => Some(range),
       UnsupportedUsing { range } => Some(range),
-      UnsupportedJavaScriptEntrypoint { range } => Some(range),
-      UnsupportedNestedJavaScript { range } => Some(range),
-      Emit(_) => None,
-      Multiple(_) => None,
+      UnsupportedJavaScriptEntrypoint { .. } => None,
+      UnsupportedNestedJavaScript { .. } => None,
+      Emit { .. } => None,
+      Multiple(_) => unreachable!("Multiple should be flattened"),
```

Comment:
Maybe debug assertion only, then return `None` in release? It's a bug, but I don't think worth taking down the process for.

Reply:
Ok, that's cool if it's going away.